tooltester/src/test/java/io/dockstore/tooltester/client/cli/ClientTest.java

```diff
@@ -119,8 +118,7 @@ public void enqueueUnverifiedTool() {
      */
     @Test
     public void createJenkinsTests() {
-        String[] argv = { "sync", "--execution", "jenkins", "--api",
-                "https://www.dockstore.org:8443/api/ga4gh/v1" };
+        String[] argv = { "sync", "--execution", "jenkins", "--api", "https://www.dockstore.org:8443/api/ga4gh/v1" };
```

Comment:
Can this go out-of-sync with https://github.com/ga4gh/dockstore-support/pull/11/files#diff-868d1ac13efe901fba1e806842cc1c08R69
What are the consequences of that?